modules/web/js/ballerina/ast/defaults-added-ballerina-ast-factory.js

```diff
@@ -168,5 +168,20 @@ define(['lodash', './ballerina-ast-factory'], function (_, BallerinaASTFactory)
         return throwStatement;
     };
 
+    /**
+     * creates ForkJoinStatement
+     * @param args
+     */
+    DefaultsAddedBallerinaASTFactory.createForkJoinStatement = function (args) {
```

Comment:
Shall we change the method name? We have the same method in ballerina-ast-factory as well.